### PR TITLE
[Merged by Bors] - feat: `a / b * (c / a) = c / b` for `CommGroupWithZero`

### PR DESCRIPTION
**Pull request by YaelDillies**
Moves:
* `div_mul_div_cancel` → `div_mul_div_cancel₀`
* `div_mul_div_cancel'` → `div_mul_div_cancel`
* `div_mul_div_cancel''` → `div_mul_div_cancel'`
* `div_div_div_cancel_right` → `div_div_div_cancel_right₀`
* `div_div_div_cancel_right'` → `div_div_div_cancel_right`

Add the missing `div_mul_div_cancel₀'`.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Algebra/Group/Basic.lean [hunk 1]
```diff
@@ -729,13 +729,13 @@ theorem div_left_inj : b / a = c / a ↔ b = c := by
...
   exact mul_left_inj _
 
-@[to_additive (attr := simp) sub_add_sub_cancel]
-theorem div_mul_div_cancel' (a b c : G) : a / b * (b / c) = a / c := by
+@[to_additive (attr := simp)]
+theorem div_mul_div_cancel (a b c : G) : a / b * (b / c) = a / c := by
   rw [← mul_div_assoc, div_mul_cancel]
 
 @[to_additive (attr := simp) sub_sub_sub_cancel_right]
 theorem div_div_div_cancel_right' (a b c : G) : a / c / (b / c) = a / b := by
```
Comment (eric-wieser) [inline Mathlib/Algebra/Group/Basic.lean:737]: Does it make sense to clean up the prime here at the same time?

Comment (YaelDillies) [reply Mathlib/Algebra/Group/Basic.lean:737]: Sure, can do

**Comment by Ruben-VandeVelde** [comment]
Looks like you're missing aliases for the names you don't reuse

**Review by jcommelin — COMMENTED** [review]
Thanks :tada:

bors merge

**Pull request closed without merging**